### ISSUE TITLE
Caffe

### ISSUE THREAD

**Issue by zamazan4ik** [conan-io/wishlist#260]
Caffe: a fast open framework for deep learning. http://caffe.berkeleyvision.org/

https://github.com/BVLC/caffe

**Comment by Croydon** [comment]
The wishlist is getting deprecated (see readme). Please open new requests in the Conan Center Index 😄

**Comment by theirix** [comment]
@ZaMaZaN4iK , I have added [conan-caffe](https://github.com/bincrafters/conan-caffe) , design  explanations are in [this ticket](https://github.com/bincrafters/community/issues/1139). It is incubating in Bincrafters repository, not in Conan Center Index yet.

**Comment by zamazan4ik** [comment]
@Croydon I am so sorry. Should I move all my previously added requests to the new place?

**Comment by Croydon** [comment]
No, it is okay having existing ones here. Just open new ones in the future in CCI 😄
